src/main/kotlin/carracing/Race.kt

```diff
@@ -1,16 +1,17 @@
 package carracing
 
+import kotlin.random.Random
+
 class Race(
-    private val numCars: Int,
+    carNames: List<String>,
     private val numRounds: Int,
 ) {
-    private val cars = List(numCars) { Car() }
+    private val cars = carNames.map { Car(it) }
```

Comment:
I'm wondering if you're familiar with the concept of First-Class Collections.
Do you think there might be any benefits to applying it to the Car objects in this case?